subsys/infinispan/src/main/resources/infinispan-jdbc.xml

```diff
@@ -0,0 +1,97 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<infinispan
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="urn:infinispan:config:8.2 http://www.infinispan.org/schemas/infinispan-config-8.2.xsd
+                        urn:infinispan:config:store:jdbc:8.0 http://www.infinispan.org/schemas/infinispan-cachestore-jdbc-config-8.0.xsd"
+    xmlns="urn:infinispan:config:8.2"
+    xmlns:jdbc="urn:infinispan:config:store:jdbc:8.0">
+
+  <cache-container default-cache="dont-use" name="IndyCacheManager" shutdown-hook="DEFAULT">
```

Comment:
we'll need to merge in the default-cache config we've added recently, so we can cap new cache counts when we forget to configure something for a new feature.